store/members/members-context.js

```diff
@@ -0,0 +1,27 @@
+import { createContext, useContext, useReducer } from 'react';
+import { membersReducer } from './members-reducer';
+import PropTypes from 'prop-types';
+
+const MembersContext = createContext({});
+const initialState = {
+  membersArr: [],
+  newMembersArr: [],
+  errorMsg: ''
+};
+
+export const MembersProvider = ({ children }) => {
+  const [state, dispatch] = useReducer(membersReducer, initialState);
+  return <MembersContext.Provider value={{ state, dispatch }}>{children}</MembersContext.Provider>;
+};
+
+export const useMembers = () => {
+  const context = useContext(MembersContext);
```

Comment:
I am using useContext here right, this is more of a custom hook created which can be accessed everywhere... That was my intention

Reply:
it's not a custom hook, it's provided by react by default 🤔

Reply:
useMembers is an custom hook created inside which I am accessing useContext which is provided by React.
Since we are using useContext here which is a react hook, I need to create custom hook useMembers right?